Public/Src/Utilities/Native/IO/Unix/FileSystem.Unix.cs

```diff
@@ -1188,35 +1187,24 @@ private bool SupportCopyOnWrite()
         /// </summary>
         public int GetFilePermission(string path, bool followSymlink = false, bool throwOnFailure = true)
         {
-            int result = GetFilePermissionsForFilePath(path, followSymlink);

```

Comment:
>GetFilePermissionsForFilePath [](start = 25, length = 29)

Should we also remove this method from the C++ side?

Reply:
It still is being referenced elsewhere.